src/platform/packages/private/kbn-language-documentation/src/components/shared/markdown_with_highlight.tsx

```diff
@@ -0,0 +1,85 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the "Elastic License
+ * 2.0", the "GNU Affero General Public License v3.0 only", and the "Server Side
+ * Public License v 1"; you may not use this file except in compliance with, at
+ * your election, the "Elastic License 2.0", the "GNU Affero General Public
+ * License v3.0 only", or the "Server Side Public License, v 1".
+ */
+
+import { getDefaultEuiMarkdownPlugins, EuiLink, EuiMarkdownFormat } from '@elastic/eui';
+import React, { useMemo } from 'react';
+
+/**
+ * Markdown component, with a plugin that supports highlighting text wrapped in ==double equals==
+ */
+export const MarkdownWithHighlight = React.memo(
+  ({
+    markdownContent,
+    openLinksInNewTab = false,
+  }: {
+    markdownContent: string;
+    openLinksInNewTab?: boolean;
+  }) => {
+    const { modifiedParsingPlugins, modifiedProcessingPlugins } = useMemo(() => {
+      const { parsingPlugins, processingPlugins } = getDefaultEuiMarkdownPlugins({
+        exclude: ['tooltip', 'lineBreaks', 'linkValidator'],
+      });
+
+      const parsingPluginList = [...parsingPlugins, highlightParsingPlugin];
+      const processingPluginList = [...processingPlugins];
+
+      // Components overwrites
+      if (openLinksInNewTab) {
+        (processingPluginList[1] as any)[1].components.a = (props: any) => (
```

Comment:
Never anys at the code, can we fix the type assertion here? If we don't know the type we can use unknown